.github/workflows/benchmark.yml

```
@@ -1,246 +1,29 @@
-          go tool pprof -svg mem.prof > mem_flame.svg || true
-      
-      - name: Upload profiling results
-        uses: actions/upload-artifact@v3
-        with:
-          name: profiling-results
-          path: |
-            benchmark/profiling/*.prof
-            benchmark/profiling/*.svg
-            benchmark/profiling/*_report.txt
-          retention-days: 30
+        uses: actions/checkout@v2
```

actionlint (1.7.10)

[error] the runner of "actions/checkout@v2" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)